pygmt/src/grdtrack.py

```diff
@@ -33,7 +34,14 @@
     n="interpolation",
 )
 @kwargs_to_strings(R="sequence", S="sequence")
-def grdtrack(points, grid, newcolname=None, outfile=None, **kwargs):
+def grdtrack(
+    points,
+    grid,
+    data_format="d",
+    df_columns=["longitude", "latitude", "z-value"],
```

Comment:
Not sure if it's a good idea to have default column names, especially for the x (longitude) and y (latitude) columns since someone passing in a `pandas.DataFrame` table with existing column names would have their column names overridden by this default.